From 23f122589ee4135a66b71a1a462682334ace0137 Mon Sep 17 00:00:00 2001
From: Abe Kohandel <abe@kodebooth.com>
Date: Mon, 3 Jul 2023 10:24:57 -0700
Subject: [PATCH] Initial patch

---
 README | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/README b/README
index 669ac7c32292..a652d92e56d8 100644
--- a/README
+++ b/README
@@ -16,3 +16,5 @@ several of them using the Restructured Text markup notation.
 Please read the Documentation/process/changes.rst file, as it contains the
 requirements for building and running the kernel, and information about
 the problems which may result by upgrading your kernel.
+
+Testing original patch
-- 
2.41.0

